CHANGES.txt

```diff
@@ -1,3 +1,7 @@
+2.0a11 (2020-05-29)
+------------------
+- refactor `rio_tiler.utils.tile_exits` to allow raster bounds latitude == -90,90
```

Comment:
Minor but I think this should be `tile_exists`

Reply:
I was pretty confused about where the tiles were going!